xl_auth/templates/401.html

```diff
@@ -5,12 +5,10 @@
 
 {% block content %}
     <div class="jumbotron">
+        <img class="img-responsive center-block" src="https://http.cat/401"><br><br>
```

Comment:
I prefer boring error messages. How about if we keep this only for the 500 template and at the same time make sure we have translations for the error messages?

Reply:
@mxtthias Sounds good! But since our dear UX team is back at work, let's have them tell us what is best.

@anjabergman @Hardesjo Är HTTP-katterna att föredra eller ser det oprofessionellt ut? Eller är det sak samma?

Hur det ser ut nu vid olika typer av fel:

![screen shot 2018-01-08 at 2 18 55 pm](https://user-images.githubusercontent.com/786326/34672672-ac79d7b8-f47f-11e7-8ab0-d759d66bbf90.png)
![screen shot 2018-01-08 at 2 16 22 pm](https://user-images.githubusercontent.com/786326/34672674-acb0fcde-f47f-11e7-85d6-d95eab524391.png)

Ett förslag som Ola och jag fann upplyftande, med bilder från [http.cat](https://http.cat):

![screen shot 2018-01-08 at 2 14 29 pm](https://user-images.githubusercontent.com/786326/34672814-31f295ec-f480-11e7-8981-08a15b270be2.png)
![screen shot 2018-01-08 at 2 10 01 pm](https://user-images.githubusercontent.com/786326/34672815-320bd0e8-f480-11e7-903e-b928261b0354.png)
![screen shot 2018-01-08 at 2 09 09 pm](https://user-images.githubusercontent.com/786326/34672816-3224030c-f480-11e7-82a5-b16e4d55cbcc.png)

![screen shot 2018-01-08 at 2 20 52 pm](https://user-images.githubusercontent.com/786326/34672847-50636c5e-f480-11e7-907f-e84063d7a54d.png)

Säg vad ni tycker!

Reply:
Missa inte att budskap i bilder behöver en alternativtext för att kunna uppfattas av personer som använder olika typer av hjälpmedel: https://webbriktlinjer.se/riktlinjer/115-textalternativ/

Reply:
@peterk Bra poäng! Fixat nu med 9920afc.